commons/utils/system_utils.py

```diff
@@ -165,7 +151,7 @@ def run_local_cmd(cmd: str = None, flg: bool = False, chk_stderr: bool = False)
     LOGGER.debug("Command: %s", cmd)
     proc = None
     try:
-        proc = Popen(cmd, shell=True, stdout=PIPE, stderr=PIPE)
+        proc = Popen(cmd, shell=False, stdout=PIPE, stderr=PIPE)
```

Comment:
Any reason why this set to False, 
default value is False only but this function is used in many files, Please make sure we are not changing mearning of execution.